### PR TITLE
Remove the textarea toggleWrap feature

### PR DESCRIPTION
**Pull request by aschempp**
Not sure we need the BC method, since someone copying the code to their own DC would probably also have the existing class (and therefore it would already work automatically) 🙃

### CODE REVIEW

File: core-bundle/src/Resources/public/toggle-wrap.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                })
+            }
+        }
+    }).observe(document, {
+        attributes: false,
+        childList: true,
+        subtree: true
+    });
+
+    Backend.toggleWrap = function (id) {
```
Comment (ausi) [inline core-bundle/src/Resources/public/toggle-wrap.js]: Should we add some checks here?

```suggestion
    if (window.Backend && !Backend.toggleWrap) {
    Backend.toggleWrap = function (id) {
```

Comment (aschempp) [reply core-bundle/src/Resources/public/toggle-wrap.js]: I would not for now. Once we can switch to Encore it all changes anyway.

**Comment by leofeyer** [comment]
I don‘t think we need the BC method. Honestly, I don‘t even think we need this feature at all. 😄

**Comment by leofeyer** [comment]
@contao/developers Do you agree that we can remove this entirely?

**Comment by aschempp** [comment]
Looks like this feature was present forever in Contao (it was in the initial commit of Contao 2.6 on GitHub 😆). I have honestly never used it, so I can also update the PR to remove it if we agree.

**Comment by ausi** [comment]
> Do you agree that we can remove this entirely?

Yes.

File: core-bundle/src/Resources/contao/classes/DataContainer.php [hunk 2]
```diff
@@ -343,12 +343,6 @@ protected function row($strPalette=null)
 
 		$xlabel = '';
 
-		// Toggle line wrap (textarea)
-		if (($arrData['inputType'] ?? null) == 'textarea' && !isset($arrData['eval']['rte']))
-		{
-			$xlabel .= ' ' . Image::getHtml('wrap.svg', $GLOBALS['TL_LANG']['MSC']['wordWrap'], 'title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['wordWrap']) . '" class="toggleWrap" onclick="Backend.toggleWrap(\'ctrl_' . $this->strInputName . '\')"');
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/classes/DataContainer.php:349]: I think we should also remove the wrap.svg icon, shouldn‘t we?

**Review by leofeyer — DISMISSED** [review]
Changes look good but you will have to resolve the conflict first.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**